examples/Instancing/main.lua

```diff
@@ -3,6 +3,10 @@
 function lovr.load()
   MONKEYS = 1000
 
+  if lovr.headset.getDriver() ~= "oculusmobile" then -- That's too many monkeys!!!
```

Comment:
Wait is this supposed to be `==`?

Reply:
Feel free to change to 700 for everyone too

Reply:
Should be enough monkeys for anyone

Reply:
It is supposed to be == :( Very confused how I made this mistake

Reply:
one second will fix